src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -82,7 +82,16 @@ function UserStats({totals, location, organization, transactionName}: Props) {
   return (
     <Container>
       <div>
-        <SectionHeading>{t('Apdex Score')}</SectionHeading>
+        <SectionHeading>
+          {t('Apdex Score')}
+          <QuestionTooltip
+            position="top"
+            title={t(
+              'Apdex is the ratio of both satisfactory and tolerable response time to all response times.'
```

Comment:
```suggestion
              'Apdex is the ratio of both satisfactory and tolerable response time with respect to all response times.'
```

Reply:
This is the same copy in another tooltip about Apdex... will do a more comprehensive tooltip review next time.